src/integration-test/java/fi/hsl/common/transitdata/MockDataUtils.java

```diff
@@ -290,7 +327,87 @@ public static void setIsViaPoint(PubtransTableProtos.Common.Builder builder, boo
                 .setStartTime(startTime)
                 .setSchemaVersion(tripCancellationBuilder.getSchemaVersion())
                 .setStatus(status);
+        if (maybeDeviationCasesType.isPresent()) {
```

Comment:
you could use map here to make it more readable. 

maybeDeviationCasesType.map(tripCancellationBuilder::setDeviationCasesType);

isPresent is useful in case you want to have an else-block but here you don't seem to use it.